### DISCUSSION TITLE
Returning data with both labels and codes

### DISCUSSION THREAD

**Discussion by rcyost** [rOpenGov/eurostat#207] [Ideas]
For example,

VA_sector_quarterly_label <- get_eurostat(id = "namq_10_a10", time_format = "num", type = "label", select_time = "Q", stringsAsFactors = FALSE )

I notice passing a vector, type = c("label", "code") does not work. 

I can't quite figure out how to request twice, with codes then labels and easily merge these two sets into one.  

Thanks

**Comment by antagomir** [comment]
Does `label_eurostat` help with this?

**Reply by rcyost** [reply]
Hmm maybe, I am not sure how I would use this function to get to my goal of a dataset of labels and codes. 

Could you advise? 

I see :

VA_sector_quarterly_label <- get_eurostat(id = "namq_10_a10", time_format = "num", type = "code", select_time = "Q", stringsAsFactors = FALSE )

labels <- label_eurostat(VA_sector_quarterly_code)

Where labels would be the dataset with no codes just labels. Is there a parameter I can use in label_eurostat() to return a dataset with both labels and codes? 

Thanks

**Reply by jhuovari** [reply]
You can use a code parameter in label_eurostat to specify columns with both labels and codes.

Option to get both using type = c("label", "code") in get_eurostat() would be a good idea.
